bayes3d/genjax/model.py

```diff
@@ -124,6 +125,8 @@ def multivmap(f, args=None):
             multivmapped = jax.vmap(multivmapped, in_axes=make_onehot(len(args), i, hot=0, cold=None))
     return multivmapped
 
+Enumerator = namedtuple("Enumerator",["enum_f", "score", "enum_f_vmap", "score_vmap"])
```

Comment:
lets brainstorm better names for these functions. Maybe `update_variables` and `update_variables_get_score` and those with `_vmap`